### PR TITLE
Set up base application and report styles

### PR DESCRIPTION
**Pull request by tomodwyer**
- 🎫 Linked to: https://github.com/opensafely-core/output-explorer/issues/66

---

- Add an EditorConfig file
- Set `DJANGO_VITE_DEV_MODE` in env file
- Set up custom styles for `notebook_contents`
- Replace Bootstrap + jQuery with Tailwind + Alpine
- Set up Inter as the application font
- Add a favicon

### CODE REVIEW

File: output_explorer/settings.py [hunk 1]
```diff
@@ -147,7 +147,7 @@
 STATIC_ROOT = BASE_DIR / "collected-static"
 
 DJANGO_VITE_ASSETS_PATH = "/static/dist/"
-DJANGO_VITE_DEV_MODE = False
+DJANGO_VITE_DEV_MODE = env.bool("DJANGO_VITE_DEV_MODE", False)
```
Comment (ghickman) [inline output_explorer/settings.py:150]: IIRC you need to use the default kwarg like `default=False` here.

File: static/manifest.webmanifest [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "OpenSAFELY",
```
Comment (ghickman) [inline static/manifest.webmanifest:2]: Should this be `OpenSAFELY Output Explorer`?

**Review by ghickman — APPROVED** [review]
Looks great, couple of small things which I think need changing.

File: output_explorer/settings.py (hunk 1 above)
Comment (tomodwyer) [reply output_explorer/settings.py:150]: Thanks, copy-pasted that from the `env` will update now.

File: static/manifest.webmanifest (hunk 2 above)
Comment (tomodwyer) [reply static/manifest.webmanifest]: 👍 Good spot

**Pull request merged**